### PR TITLE
Adding affiliated package support

### PR DESCRIPTION
**Pull request by henryiii**
Packages can now be marked affiliated. They will show up like this:

Screen Shot 2020-01-02 at 4 14 55 PM

Suggestions for emoji welcome. They are also integrated into the docs page, and the affiliated page is generated. Several other smaller updates.

Also includes small logos on the main page. Can eventually be turned into cards when all projects have logos.

### CODE REVIEW

File: _data/projects/event/awkward-array.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 name: awkward-array
+logo: /assets/images/projlogo/logo_awk.png
```
Comment (eduardo-rodrigues) [inline _data/projects/event/awkward-array.yml]: I would have written `projlogos`, but not a big deal at all.

Comment (eduardo-rodrigues) [reply _data/projects/event/awkward-array.yml]: For such reasonably short package names, why not write them in full? Here and in a couple of other examples?

**Comment by eduardo-rodrigues** [comment]
For the emoji for affiliated packages one could have a hand-in-hand logo, so kind of a shake-hand? Seems appropriate for affiliated packages, which in some sense are packages that go hand-in-hand.

**Comment by henryiii** [comment]
🤝?

**Comment by eduardo-rodrigues** [comment]
> 🤝?

Yeah, sounds good, at least for a first version.

**Comment by henryiii** [comment]
How's that?

**Comment by eduardo-rodrigues** [comment]
To be honest I think AmpGen is on the edge, since not at all a Python package. Also not using anthing from Scikit-HEP, though that's not really an issue since the concept of affiliated is about an ecosystem for HEP. But I let you decide, as I have no strong opinion.

**Comment by henryiii** [comment]
I assumed going from CamelCase DecayLanguage to lowercase would be decay_language...

**Comment by henryiii** [comment]
AmpGen does have a Python interface to the generated output SO's that I added a year ago or so, which is why I included it. Doesn't have to be there, and doesn't have to go in now.

**Comment by eduardo-rodrigues** [comment]
> I assumed going from CamelCase DecayLanguage to lowercase would be decay_language...

That's fine. I was in the mindset of writing all names lowercase, simply. OK, anyway, as you wrote.

**Comment by eduardo-rodrigues** [comment]
> AmpGen does have a Python interface to the generated output SO's that I added a year ago or so, which is why I included it. Doesn't have to be there, and doesn't have to go in now.

Oh, I did not know that. OK.

**Comment by henryiii** [comment]
I'm using lower case with no `_` now for the filenames, since that's the import name.

**Comment by henryiii** [comment]
I've dropped AmpGen, we can add it later. Probably should check with Tim first.

**Comment by eduardo-rodrigues** [comment]
> I've dropped AmpGen, we can add it later. Probably should check with Tim first.

I think so.

**Comment by henryiii** [comment]
I'm ready for it to go in when you are.

Linking to an "Awesome HEP list" might be useful here eventually, as we would like to highlight packages that are useful but not "affiliated", as in, we have no direct affiliation with the authors.

**Comment by henryiii** [comment]
Conda-forge ROOT might be a nice thing to add at some point, too. Maybe after/during #29.

**Comment by eduardo-rodrigues** [comment]
Agree on both points. Will come back to it soon.

Ready to go, I reckon.

**Pull request merged**